docs/apache-spark-with-minio.md

```diff
@@ -149,21 +136,33 @@ You can create, read, and write Delta tables on MinIO using the `s3a` connector.
 --conf spark.hadoop.fs.s3a.impl=org.apache.hadoop.fs.s3a.S3AFileSystem
 ```
 
-Here, we're using the Delta Lake release `0.5.0` with Scala `2.11` release, while the `s3a` fields based on the MinIO cluster. After you run this and get the Spark console, you can run delta and Spark APIs. Refer Delta Lake [storage configuration docs](https://docs.delta.io/latest/delta-storage.html) here.
-
-### Write Test
-
-To test if you can use the delta API, use the below commands in spark console
+Here, we're using the Delta Lake release `0.5.0` with Scala `2.11` release, while the `s3a` fields based on the MinIO cluster. After you run this and get the Spark console, you can run delta and Spark APIs. Refer Delta Lake [storage configuration docs](https://docs.delta.io/latest/delta-storage.html) here. To test if you can use the delta API, use the below commands in spark console
 
 ```sh
 spark.range(5).write.format("delta").save("s3a://spark-test/delta")
 ```
 
-### Read Test
-
 To check if the reads are working fine, attempt reading the file we created in write test.
 
 ```sh
 val df = spark.read.format("delta").load("s3a://spark-test/delta")
 df.show()
 ```
+
+## 7. Configuration Notes
+
+### MinIO server configured with HTTPS
+
+In case you are planning to use self-signed certificates for testing purposes, you'll need to add these certificates to local JRE `cacerts`. You can do this using
+
+```sh
+keytool -import -trustcacerts -keystore $JAVA_HOME/jre/lib/security/cacerts -storepass changeit -noprompt -alias mycert -file /home/username/.minio/certs/public.crt
+```
+
+Also, change the scheme to `https` in `spark.hadoop.fs.s3a.endpoint` field while passing to `spark-shell`.
+
+### MinIO server with custom region
+
+If you're using MinIO with a custom region (instead of default region `us-east-1`), you'll need to use endpoint with relevant region added. For example, if your MinIO server is running with region `ap-southeast-1` at `http://localhost:9000`, then `spark.hadoop.fs.s3a.endpoint` flag should be set to `http://s3.ap-southeast-1.localhost:9000`.
```

Comment:
```suggestion
If you are using MinIO with a custom region (instead of default region `us-east-1`), you'll need to use endpoint with relevant region added. For example, if your MinIO server is running with region `ap-southeast-1` at `http://localhost:9000`, then `spark.hadoop.fs.s3a.endpoint` flag should be set to `http://s3.ap-southeast-1.localhost:9000`.
```